cmd/connector/main.go

```diff
@@ -54,7 +57,17 @@ func main() {
 }
 
 func startConnector(ctx *cli.Context) error {
-	cfg, err := loadConfig(configPath)
+	configFilePath := ctx.GlobalString(configFile.Name)
+	if configFilePath == "" {
+		configFilePath = defaultConfigPath
+	}
```

Comment:
default config path can be set as `Value` for `cli.StringFlag` and there is no need for this additional check

Reply:
done.